stats_config.yaml

```diff
@@ -128,3 +128,15 @@
   query: .logGroups|length
   source: logs-describe-log-groups
   verbose: true
+- name: ACM Certificates 
+  query: .CertificateSummaryList|length
+  source: acm-list-certificates  
+  verbose: true
+- name: DynamoDB Tables 
+  query: .TableNames|length
+  source: dynamodb-list-tanñes  
```

Comment:
I believe there's a typo here.
tables right?

Reply:
Yes, I have fixed it!